src/sbvr-api/sbvr-utils.coffee

```diff
@@ -42,6 +42,8 @@ memoizedCompileRule = memoize(
 
 db = null
 
+exports.hashApiKey = permissions.hashApiKey
```

Comment:
Everything from `permissions` is already exported below via an `_.extend(exports, permissions)`